importers/src/bible/NEUE/ref-parser-test.ts

```diff
@@ -7,17 +16,35 @@ bcv.set_options({
     invalid_passage_strategy: 'include',
     invalid_sequence_strategy: 'include',
     passage_existence_strategy: 'bc',
+    consecutive_combination_strategy: 'separate',
 });
 const localRefRegEx = /(Kapitel|V\.|Vers) ([0-9,.\-–; ]|(und|bis|Kapitel|V\.|Vers))+/g;
+// const localRefRegEx = /(^| )(chapter|ch\.|v\.|verse|verses) ([0-9:\-–; ,]|(and|to|chapter|ch\.|v\.|verse|verses))+/gi;
 
 // const testString =
 //     'Vgl. insbesondere Vers 5 und 11 und Kapitel 2 mit 3 und Joh 3,16 und Kapitel 7,2';
 
 // const testString = 'was auch zu 1. Mose 10,8-12 passen würd';
-const testString = 'abc Jesaja 8,23 – 9,1 sdf';
+// const testString = 'abc Jesaja 8,23 – 9,1 sdf';
+// const testString = 'Vergleiche die Anmerkungen zu Kapitel 14,7 und 16,7.';
+// const testString = 'canopy; also verses 7, 8, 14, 15, 17, 20';
+// const testString = 'Hebrew verse 5';
+// const testString =
+// 'v. 13 ir neglect of his temple (see Lev. 26:2-20). He called them in v. 1 to repent and renew their covenant with the God of th';
+// const testString =
+// 'bezeichnet sowohl den Lufthimmel über der Erde (z.B. 5Mo 4,17) als auch den Sternenhimmel (z.B. 5Mo 4,19) und den Himmel als Thronsitz Gottes (z.B. Ps 2,4).';
+// const testString =
+//     ' das, was Gott heilig ist, vor Mißbrauch und Entweihung zu schützen (vgl. u.a. 2Mo 25,18; 26,1; 1Kön 8,6).';
```

Comment:
yes, nice to have. however since is currently just a playground file. since it's possibly useful later i leave it in the repo. proper test is something for a later time.